src/sentry/static/sentry/app/components/projectSelector.jsx

```diff
@@ -180,20 +185,19 @@ class ProjectSelector extends React.Component {
         maxHeight={500}
         zIndex={theme.zIndex.dropdown}
         css={{marginTop: 6}}
-        inputProps={{style: {padding: 8, paddingLeft: 14}}}
+        inputProps={{style: {padding: 8, paddingLeft: 10}}}
         rootClassName={rootClassName}
         className={className}
         emptyMessage={t('You have no projects')}
         noResultsMessage={t('No projects found')}
         virtualizedHeight={theme.headerSelectorRowHeight}
         emptyHidesInput
         inputActions={() => (
-          <div>
-            <ManageButton to={`/settings/${org.slug}/projects/`} size="xsmall">
-              <StyledSettingsIcon src="icon-settings" />
-              {t('Manage')}
-            </ManageButton>
-          </div>
+          <Tooltip title="Add a project">
+            <AddButton to={`/organizations/${org.slug}/projects/new`} size="xsmall">
+              <StyledAddIcon src="icon-circle-add" /> project
```

Comment:
Can we do the disable in a follow up along with translate + capitalizing `Project`?

Reply:
yeah i have the disabling functionality plus a new "update" button stashed and ready to go for a follow up.